package.json

```diff
@@ -2,7 +2,7 @@
   "name": "bodhi-ui",
   "version": "1.0.4",
   "dependencies": {
-    "@material-ui/core": "^1.2.2",
+    "@material-ui/core": "latest",
```

Comment:
use static version number: `^3.2.2`. if we always point to the latest, the whole UI can break without us knowing what broke it if they make breaking changes in material-ui.